packages/katello/katello/hostname-change.rb

```diff
@@ -345,6 +475,11 @@ def run
       STDOUT.puts "updating hostname in foreman installer scenarios"
       self.run_cmd("sed -i -e 's/#{@old_hostname}/#{@new_hostname}/g' #{scenarios_path}/*.yaml")
 
+      if @scenario_answers['foreman_proxy']['dns']
+        dns_data = get_dns_data
+        update_dns_records(dns_data)
```

Comment:
This variable assignment seems unnecessary

Reply:
@jturel It will be once we come up with a better name for `get_dns_data` :yum: